### ISSUE TITLE
htiled does not load "Collection of Images"

### ISSUE THREAD

**Issue by seppeljordan** [chrra/htiled#7]
htiled expects something like this:

```xml
<tileset firstgid="3" name="darkBricks" tilewidth="32" tileheight="32">
    <image source="darkBricks.bmp" trans="ff00ff" width="256" height="256"/>
</tileset>
```

where the `image` node is a child of a `tileset` node.

When a user creates a tileset in Tiled that is based on a collection of images, tiled will generate the following structure

```xml
<tileset firstgid="1" name="tileset2" tilewidth="36" tileheight="36">
    <terraintypes>
        <terrain name="Bla" tile="-1"/>
    </terraintypes>
    <tile id="0" terrain="0,0,0,0">
        <image width="36" height="36" source="redBlock.bmp"/>
    </tile>
    <tile id="1" terrain="0,0,0,0">
        <properties>
            <property name="trans" value="0"/>
        </properties>
        <image width="15" height="15" source="darkRedSquare.bmp"/>
    </tile>
</tileset>
```

The xml parser does not recognize this structure and does not parse the images.

**Comment by m09** [comment]
Merged: https://github.com/chrra/htiled/pull/8. Thanks for reporting it and providing a patch.

**Comment by m09** [comment]
Available in versino 0.1.3.0: https://hackage.haskell.org/package/htiled-0.1.3.0
